tests/test_io/test_spikeglx.py

```diff
@@ -106,8 +106,8 @@ def test_NP2_4_shanks_with_different_electrodes_saved():
 
     # This file does not save the channnels from 0 as the one above (NP2_4_shanks_g0_t0.imec0.ap.meta)
     ypos = probe.contact_positions[:, 1]
-    assert np.min(ypos) == pytest.approx(2880.0)
-    assert np.max(ypos) == pytest.approx(5745.0)
+    assert np.min(ypos) == pytest.approx(4080.0)
+    assert np.max(ypos) == pytest.approx(4785.0)
```

Comment:
Bad boy, you cheat : you are changing the test until it fit the change you made :)
But the bug was also in the test so do not trust test with magic number in.